.github/workflows/build-devcontainer-image.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       contents: read
 
     steps:
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
The issue was that somehow `setup-buildx-action` is actually not needed and breaks the `build-push-action` below?

Reply:
Seems to be. I reverted back to the version that worked prior to Apr 26 and made changes one by one to move it forward. 

According to the docs, the `setup-buildx-action` is optional, but recommended. Leaving it out seems to be the key though.